core/src/main/scala/scalaz/std/String.scala

```diff
@@ -6,7 +6,7 @@ trait StringInstances {
     type SA[α] = String
     def append(f1: String, f2: => String) = f1 + f2
     def zero: String = ""
-    override def show(f: String) = '"' + f + '"'
+    override def shows(f: String): String = f
```

Comment:
Notice the change from `show` to `shows`.

Reply:
I don't think it's a good idea to change behavior like this in 7.2.x. I do agree that this is how it should've been from the start, but that's beside the point.